supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchExpressionFunctions.scala

```diff
@@ -49,3 +49,8 @@ class AwsBatchExpressionFunctions(standardParams: StandardExpressionFunctionsPar
     }
   }
 }
+
+class AwsBatchExpressionFunctionsForFS(standardParams: StandardExpressionFunctionsParams)
```

Comment:
Shoud this be `ForEFS`?

Reply:
It will be the same for any mountable shared FileSystem, not just EFS. That why I kept it as FS.